src/freenas/usr/local/libexec/freenas-debug/smb/smb.sh

```diff
@@ -124,8 +118,8 @@ smb_func()
 	section_header "net getdomainsid"
 	net getdomainsid
 	section_footer
-	section_header "net groupmap list"
-	net groupmap list | head -50
+	section_header "net --json groupmap list"
```

Comment:
I remember this potentially barfing tons of information which is why we limited it to 50 entries. You concerned with the changes here?

Reply:
Oh, right, that's when we have ldap enabled. Will fix.

Reply:
When we use `smb.groupmap_list` and Samba's passdb backend is ldapsam, we will skip this.